cypress/integration/no_role/no_role.js

```diff
@@ -0,0 +1,45 @@
+import { Given, When, Then, Before, After, And } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@test.no';
+const NO_ROLE = '';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, NO_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+
+      cy.setLanguage();
+      cy.visit('/');
+    });
+  });
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@userId');
+});
+And('they have no NVA role', () => {});
+
+When('they look at any page in NVA', () => {
+  cy.visit('/');
+});
+
+Then('they see a menu containing', (tableData) => {

```

Comment:
Can the check method here be extracted as a function as we're doing the same thing multiple times?